### DISCUSSION TITLE
Why doesn't Outliner work in preview mode?

### DISCUSSION THREAD

**Discussion by Bennett-Yang** [vslinko/obsidian-outliner#112] [Q&A] [answered]
Outliner works perfectly in edit mode and I really love it. However, I find it does not work in preview mode. Am I missing something or it just does not have this function?

**Comment by vslinko** [comment, marked as answer]
@Bennett-Yang Yes, the plugin doesn't have features in preview mode yet.

There is one feature request #66. Please create more feature requests if you have ideas.

**Reply by Bennett-Yang** [reply]
Ok, thank you!
